src/components/DataDictionary.tsx

```diff
@@ -142,7 +146,11 @@ export const columns: ColumnDescriptor[] = [
     label: "Tags",
     rowValueDescriptor: "tags",
     formatter: (tags: any) => {
-      return tags.map((tag: string) => <Chip disabled>{tag}</Chip>)
+      return tags.map((tag: string) => (
```

Comment:
no need to `return` if it's the only thing happening in an arrow function. However, you must return arrays wrapped inside `React.Fragment` (or `<>`, its syntactic sugar):

```typesript
(tags: string[]) => (
  <>
    {tags.map(tag => <Chip disabled key={tag}>{tag}</Chip>)}
  </>
)
```

Reply:
It's a bit of a problem that the first argument to formatter can be `string | string[]`